{
  "Class": "vulnerability",
  "ID": "oval:com.redhat.cve:def:20243651",
  "Version": "636",
  "Metadata": {
    "Title": "python-idna: potential DoS via resource consumption via specially crafted inputs to idna.encode() (moderate)",
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2024-3651",
        "RefURL": "https://access.redhat.com/security/cve/CVE-2024-3651"
      }
    ],
    "Description": "DOCUMENTATION: A flaw was found in the python-idna library. A malicious argument was sent to the idna.encode() function can trigger an uncontrolled resource consumption, resulting in a denial of service.",
    "Advisory": {
      "From": "secalert@redhat.com",
      "Severity": "Moderate",
      "Issued": {},
      "Updated": {
        "Date": "2024-04-18"
      },
      "Cves": [
        {
          "CveID": "CVE-2024-3651",
          "Cvss3": "6.5/CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
          "Cwe": "CWE-400",
          "Impact": "moderate",
          "Href": "https://access.redhat.com/security/cve/CVE-2024-3651",
          "Public": "20240412"
        }
      ],
      "AffectedCpeList": [
        "cpe:/a:redhat:enterprise_linux:9",
        "cpe:/a:redhat:enterprise_linux:9::appstream",
        "cpe:/a:redhat:enterprise_linux:9::crb",
        "cpe:/a:redhat:enterprise_linux:9::highavailability",
        "cpe:/a:redhat:enterprise_linux:9::nfv",
        "cpe:/a:redhat:enterprise_linux:9::realtime",
        "cpe:/a:redhat:enterprise_linux:9::resilientstorage",
        "cpe:/a:redhat:enterprise_linux:9::sap",
        "cpe:/a:redhat:enterprise_linux:9::sap_hana",
        "cpe:/a:redhat:enterprise_linux:9::supplementary",
        "cpe:/o:redhat:enterprise_linux:9",
        "cpe:/o:redhat:enterprise_linux:9::baseos"
      ],
      "Affected": {
        "Resolution": {
          "State": "Under investigation"
        }
      }
    }
  },
  "Criteria": {
    "Operator": "OR",
    "Criterions": [
      {
        "Comment": "Red Hat Enterprise Linux must be installed",
        "TestRef": "oval:com.redhat.cve:tst:20052541004"
      }
    ],
    "Criterias": [
      {
        "Operator": "AND",
        "Criterions": [
          {
            "Comment": "Red Hat Enterprise Linux 9 is installed",
            "TestRef": "oval:com.redhat.cve:tst:20052541003"
          },
          {
            "Comment": "python3-idna is installed",
            "TestRef": "oval:com.redhat.cve:tst:20243651001"
          },
          {
            "Comment": "python3-idna is signed with Red Hat redhatrelease2 key",
            "TestRef": "oval:com.redhat.cve:tst:20243651002"
          }
        ]
      }
    ]
  }
}